web/app/themes/ppj/src/vue/JobSummary.vue

```diff
@@ -73,7 +73,26 @@
       },
 
       formattedSalary: function() {
-        return '£' + Intl.NumberFormat().format(this.salary)
+
+        if(isNaN(this.salary)) {
```

Comment:
There are a lot of _if_ statements inside _if_ statements here, which is maybe a bit confusing to someone who didn't know what this was doing but I think its okay.